src/core/transport/ntcp_session.cc

```diff
@@ -251,16 +251,16 @@ void NTCPSession::SendPhase2() {
       m_Establisher->phase2.encrypted.hxy.data(),
       xy.data(),
       static_cast<std::size_t>(NTCPSize::pub_key) * 2);
-  std::uint32_t tsB = htobe32(i2p::util::GetSecondsSinceEpoch());
-  m_Establisher->phase2.encrypted.timestamp = tsB;
+  std::uint32_t ts_b = htobe32(i2p::util::GetSecondsSinceEpoch());
```

Comment:
`b` can remain uppercase as we need to differentiate between timestamp A and B. Even better, rename to `timestamp_B`.